frontend/js/src/common/brainzplayer/FunkwhalePlayer.tsx

```diff
@@ -230,20 +230,26 @@ export default class FunkwhalePlayer
   };
 
   getArtistNamesFromTrack = (track: FunkwhaleTrack): string => {
-    if (!track.artist_credit || track.artist_credit.length === 0) {
-      return "";
+    // Handle API format with artist_credit
+    if (track.artist_credit && track.artist_credit.length > 0) {
```

Comment:
We can make that a bit more compact:

```suggestion
    if (track.artist_credit?.length) {
```